demos/toolbar/index.html

```diff
@@ -33,9 +33,24 @@
       <h2>Normal Toolbar <button type="button" onclick="toggleRTL(0)">Toggle RTL</button></h2>
       <p><a href="./default-toolbar.html" target="_blank">View in separate window</a></p>
       <iframe class="demo-container" src="./default-toolbar.html"></iframe>
-      <h2>Fixed Toolbar <button type="button" onclick="toggleRTL(1)">Toggle RTL</button></h2>
+      <h2>Waterfall Toolbar<button type="button" onclick="toggleRTL(1)">Toggle RTL</button></h2>
       <p><a href="./fixed-top-toolbar.html" target="_blank">View in separate window</a></p>
       <iframe class="demo-container" src="./fixed-top-toolbar.html"></iframe>
+      <h2>Waterfall Toolbar (CSS Only)<button type="button" onclick="toggleRTL(2)">Toggle RTL</button></h2>
```

Comment:
Fixed toolbar is _always_ fixed and _always_ has elevation.

Waterfall toolbar is _initially_ static and has no elevation, and then when the user starts scrolling _becomes_ fixed and _gains_ elevation.

Thus I think it makes sense to split it into two sections.